packages/services/document-service/src/main/java/software/uncharted/terarium/documentserver/resources/xdd/SetResource.java

```diff
@@ -1,29 +1,44 @@
 package software.uncharted.terarium.documentserver.resources.xdd;
 
+import io.quarkus.cache.CacheInvalidate;
+import io.quarkus.cache.CacheResult;
+import lombok.extern.slf4j.Slf4j;
 import org.eclipse.microprofile.openapi.annotations.tags.Tag;
 import org.eclipse.microprofile.rest.client.inject.RestClient;
 import software.uncharted.terarium.documentserver.responses.xdd.XDDSetsResponse;
 import software.uncharted.terarium.documentserver.proxies.xdd.DocumentProxy;
 
-import javax.ws.rs.Consumes;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
+import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
 
 @Path("/sets")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
 @Tag(name = "XDD Sets REST Endpoint")
+@Slf4j
 public class SetResource {
 
+	private static final String CACHE_NAME = "xDD-Sets";
```

Comment:
I would move these to an enum so you can do grouped cached clearing. Eg:
```java
public enum CacheCollection {
	XDD
}

public enum CacheName {
	XDD_SETS("xdd-sets", XDD),
	XDD_DICTIONARIES("xdd-dictionaries", XDD);


	@Getter
	private String name;
	@Getter
	private CacheCollection collection;

	CacheName(String name, CacheCollection collection) {
		this.name = name;
		this.collection = collection;
	}
	
	public static List<CacheName> findAllByCollection(CacheCollection collection) {
		return Stream.of(values())
			.filter(cacheName -> cacheName.collection.equals(collection))
			.collect(Collectors.toList());
	}
}
```

Then a simple scheduled method:
```java
@ApplicationScoped
public class CacheClearService {
  @Inject
  CacheManager cacheManager;

  @Scheduled(cron="0 * * * *") // every hour on the hour
  void clearXDDCaches(ScheduledExecution execution) {
        CacheNames.findAllByCollection(XDD).forEach(cacheName -> {
           cacheManager.clear(cacheName.getName());
        }
    }
```

Reply:
Updated.  There are some rules about using enums in annoation names that required the Strings to be stored separately, however, it is still way less fragile.